packages/io/src/types/index.ts

```diff
@@ -57,11 +57,10 @@ export type RoutePayload<
   Body extends object = object,
 > = {
   query?: Query
-  body?: Body
-  params?: Params
   path: Path
   method: Method
-}
+} & ([Params] extends [never] ? Record<string, never> : { params: Params }) &
```

Comment:
AFAIK, `never` should be used as a `response` of a function only. In this case, what does `Record<string, never>` mean

Reply:
`Record<string, never>` means `{}`

Reply:
Cool, got new knowledge